src/approval/approvers.interface.ts

```diff
@@ -0,0 +1,28 @@
+import { IUser } from "../utils/user.interface";
+
+export interface IApproverInfo {
+    userId: string;
+    isAdmin: boolean;
+    isApprover: boolean;
+    isBlocked: boolean;
+    unit: {
+        name: string;
+        approvers: string[];
+    };
+}
+
+export interface ICanApproveToUser {
+    canApproveToUser: boolean;
+    cantApproveReasons?: string[];
+}
+
+export interface IApprovalRequest {
```

Comment:
`ApprovalRequest` sounds like someone is requesting approval. I don't think it's the purpose of the object.

Reply:
Does IRequest sounds better?

Reply:
Well... I don't know 😞 
I guess it doesn't really matter that much.